src/pages/SavedArticles.jsx

```diff
@@ -1,24 +1,57 @@
-import React, { useEffect } from 'react';
+import React, { useEffect, useContext, useState } from 'react';
 import NewsCardList from '../components/NewsCardList/NewsCardList.jsx';
+import CurrentUserContext from '../contexts/CurrentUserContext';
+import CardType from '../constants/enums/CardType';
+
+export default function SavedArticles({ savedCards, onCardRemoveClick }) {
+  const currentUser = useContext(CurrentUserContext);
+  const keywords = [];
+  const [keywordText, setKeywordText] = useState('');
 
-export default function SavedArticles() {
-  // const [loggedIn, setLoggedIn] = React.useState(false);
   useEffect(() => {
     const app = document.querySelector('.app');
```

Comment:
**Might be better**

`document.querySelector` is not allowed to use in `React` because it’s a **very slow** operation for the browser. React was specially created not to search anything in the DOM. So, you can directly operate with the markup inside the components. Or you can use hook `useRef` for the reference.

or you can pass `setAppStyles` to the props and set it here


```js

  useEffect(() => {
    setAppStyles('app_page_saved-articles')
    return () => {
      setAppStyles('')
    };
  }, []);

```